Source/Editor/SceneGraph/Actors/SplineNode.cs

```diff
@@ -298,6 +303,11 @@ public SplineNode(Actor actor)
 
         private unsafe void OnUpdate()
         {
+            if (Input.Keyboard.GetKey(KeyboardKeys.Shift))
```

Comment:
I have used Shift, this can be changed in the future.

I tried using Control but this comflicts with TransformGizmo snap, using Shift has a small problem, if you press Shift before start move selected spline point has ben duplicated (Duplicate point shortcut is shift key too....)